oleesir/authors-haven-api
# #168675497 Setup ESLint

#### What does this PR do?
This PR implements the endpoint Setup ESLint
#### Description of Task to be completed?
Set up ESLint
#### How should this be manually tested?
- clone the git repo and run 'npm install' to download the necessary dependencies.
- ESLint will help check for style violations and flag them.
#### What are the relevant pivotal tracker stories?
[#168675497](https://www.pivotaltracker.com/story/show/168675497)
#### Any background context you want to add?
N/A
#### Screenshots
N/A
